pyvista/plotting/themes.py

```diff
@@ -33,10 +33,12 @@
 from enum import Enum
 import json
 import os
+import pathlib
 from typing import Callable, List, Optional, Union
 import warnings
 
 import pyvista
+from pyvista import examples
```

Comment:
Hm. This might have implications... would prefer to not have this cross import

Reply:
Setting the default value to `None` is maybe enough because of the complexity of cross-importing.